clients/admin-ui/src/features/user-management/form/inputs.tsx

```diff
@@ -0,0 +1,78 @@
+// Adapted from https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/features/common/form/inputs.tsx
+// Should in the future consider pulling this into fidesui
+
+import {
+  Box,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  IconButton,
+  Input,
+  InputGroup,
+  InputRightElement,
+} from "@fidesui/react";
+import { FieldHookConfig, useField } from "formik";
+import { useState } from "react";
+
+import { EyeIcon } from "../../common/Icon";
+
+interface InputProps {
```

Comment:
Is our standard to use interface or type?

Reply:
oh I don't know! I use `interface` out of habit, but it appears they're [pretty interchangeable](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces)? happy to go with whatever people prefer